jenkins/webhook-proxy/main.go

```diff
@@ -428,11 +428,18 @@ func (c *ocClient) Forward(e *Event, triggerSecret string) ([]byte, error) {
 	)
 	log.Println(e.RequestID, "Forwarding to", url)
 
-	req, _ := http.NewRequest(
-		"POST",
-		url,
-		new(bytes.Buffer),
-	)
+	p := struct {
+		Env []EnvPair `json:"env"`
+	}{
+		Env: e.Env,
+	}
+	b := new(bytes.Buffer)
+	err := json.NewEncoder(b).Encode(p)
+	if err != nil {
+		return nil, fmt.Errorf("Could not encode payload: %s", err)
+	}
+
+	req, _ := http.NewRequest("POST", url, b)
```

Comment:
@michaelsauter  and that works? updating an existing pipeline with a post?

Reply:
Yes. See the original issue. The behaviour is documented at https://docs.openshift.com/container-platform/3.11/dev_guide/builds/triggering_builds.html#generic-webhooks.

It does not update the pipeline object, it sets the params on the build object.